Configurations.md

```diff
@@ -2680,6 +2732,17 @@ By default this option is set as a percentage of [`max_width`](#max_width) provi
 
 See also [`max_width`](#max_width), [`use_small_heuristics`](#use_small_heuristics), and [`struct_lit_single_line`](#struct_lit_single_line)
 
+## `style_edition`
+
+Controls the edition of the [Rust Style Guide] to use for formatting ([RFC 3338])	
+
+- **Default value**: `"2015"`	
+- **Possible values**: `"2015"`, `"2018"`, `"2021"`, `"2024"` (unstable variant)	
+- **Stable**: No	
+
+[Rust Style Guide]: https://doc.rust-lang.org/nightly/style-guide/	
+[RFC 3338]: https://rust-lang.github.io/rfcs/3338-style-evolution.html	
+
```

Comment:
Please revert this change. I don't want to make any changes to other config documentation in this PR.

Reply:
fixed. it was caused by a merge conflict